### DISCUSSION TITLE
When saving pdf page to png, how to determine dpi automatically for each page like Adobe?

### DISCUSSION THREAD

**Discussion by nissansz** [pymupdf/PyMuPDF#2129] [Ideas]
When saving pdf page to png, how to determine dpi automatically for each page like Adobe?

![image](https://user-images.githubusercontent.com/41010669/208003703-dee70a8f-5bc1-4cb3-b49a-e05987cfa42e.png)

**Comment by JorjMcKie** [comment]
What's their algorithm?

**Comment by nissansz** [comment]
I don't know, you can try to use Adobe Acrobat.

**Reply by JorjMcKie** [reply]
Not interested: Seems to be a non-free feature.

**Comment by nissansz** [comment]
No simply way?

**Reply by JorjMcKie** [reply]
Don't know one - would have to guess like you.
